.env

```diff
@@ -17,7 +17,7 @@ COMPOSE_FILE=docker-compose.yml
 ###### For Tests or local development ######
 #COMPOSE_FILE=docker-compose-for-tests.yml
 # To run all additional integrations in development
-#COMPOSE_FILE=docker-compose-for-tests.yml:./integrations/docker-compose-for-tests.peframe.yml:./integrations/docker-compose-for-tests.thug.yml:./integrations/docker-compose-for-tests.capa.yml:./integrations/docker-compose-for-tests.boxjs.yml
+COMPOSE_FILE=docker-compose-for-tests.yml:./integrations/docker-compose-for-tests.capa.yml:./integrations/docker-compose-for-tests.floss.yml
```

Comment:
you forgot to do not push the change on this file

Reply:
Oh, I'll update.